src/contentTypeConfig.js

```diff
@@ -227,6 +227,13 @@ let contentTypeConfig = {
            `"description": ${descriptionField}`,
            '"instructors": instructor[]->name'
         ],
+    },
+    'routine':{
+        'fields': [
+            'description',
```

Comment:
Description is a block field and has a strange structure. We may change the Sanity behaviour later, so using the constant should make it more future proof.
```suggestion
            `"description": ${descriptionField}`,
```